plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -708,7 +708,7 @@ global proc mayaUsdMenu_duplicateToUSD( string $proxy, string $obj )
     }
 
     for ($item in $bulkObjects) {
-        string $exportOptions = `python("import mayaUsdDuplicateAsUsdDataOptions; import mayaUsdOptions; mayaUsdOptions.setAnimateOption('''" + $obj + "''', mayaUsdDuplicateAsUsdDataOptions.getDuplicateAsUsdDataOptionsText())")`;
+        string $exportOptions = `python("import mayaUsdDuplicateAsUsdDataOptions; import mayaUsdOptions; mayaUsdOptions.setAnimateOption('''" + $item + "''', mayaUsdDuplicateAsUsdDataOptions.getDuplicateAsUsdDataOptionsText())")`;
```

Comment:
Is obj the parent node? and if so, do we also need to setAnimateOption on that?
Was item ever used before this change?

Reply:
$item is replacing $obj, but in the original change I forgot to change it here.